client/dive-common/components/Viewer.vue

```diff
@@ -380,14 +404,14 @@ export default defineComponent({
     loadData();
 
     const reloadAnnotations = async () => {
-      clearAllTracks();
+      trackStore.clearAll();
```

Comment:
Wanted to log this ahead of time, the `groupStore` needs to be cleared here as well as the `trackStore`.